packages/extension-sdk/src/connect/extension_host_api.ts

```diff
@@ -52,54 +54,72 @@ export class ExtensionHostApiImpl implements ExtensionHostApi {
   private chattyHost: ChattyHostConnection
   private setInitialRoute?: (route: string, routeState?: any) => void
   private hostChangedRoute?: (route: string, routeState?: any) => void
+  private vizualizationDataReceivedCallback?: VizualizationDataReceivedCallback
+
   private contextData?: string
 
   constructor(configuration: ExtensionHostApiConfiguration) {
     this._configuration = configuration
-    const { chattyHost, setInitialRoute, hostChangedRoute } =
-      this._configuration
+    const {
+      chattyHost,
+      setInitialRoute,
+      hostChangedRoute,
+      vizualizationDataReceivedCallback,
```

Comment:
Bit by English English vs. American English confusion here, I bet.

```suggestion
      visualizationDataReceivedCallback,
```

Reply:
Dammit! I keep wanting to type viz! Fixed